public/index.html

```diff
@@ -10,34 +10,13 @@
       content="Web site created using create-react-app"
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
-    <!--
-      manifest.json provides metadata used when your web app is installed on a
-      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
-    -->
+
```

Comment:
Nice addition of the icon! Could it be renamed to something less generic? Also are both of the icon files uploaded necessary?